src/components/Profile/Presentation.tsx

```diff
@@ -73,8 +73,29 @@ const Presentation: React.SFC<IUserProfile & IProps> = props => {
       <Content className="rvt-bg-white">
         <Row className="rvt-p-top-lg">
           <Col md={6} className="rvt-p-all-lg">
-            <Panel title="Personal Information">
-              <div className="list-dividers">
+            <Panel title="Contact Information">
+            {user.location && 
+                <div>
+                  <strong>Location:</strong> 
```

Comment:
Need to a space after the colon
```suggestion
                  <strong>Location:</strong>{" "}
```

Or even remove the label, since the format of the could indicate its type
```suggestion
```

Reply:
This would apply to add the contact information

Reply:
current rendering looks like this:
<img width="479" alt="screen shot 2018-12-12 at 7 46 46 pm" src="https://user-images.githubusercontent.com/862054/49910244-8f977100-fe47-11e8-9bba-593c71f58aac.png">